src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -163,20 +174,24 @@ void LiveFunctionsDataView::UpdateHistogramWithIndices(
                  std::back_inserter(function_ids),
                  [this](const int index) { return indices_[index]; });
 
-  UpdateHistogramWithFunctionIds(function_ids);
+  UpdateHistogramWithScopeIds(function_ids);
 }
 
-void LiveFunctionsDataView::UpdateHistogramWithFunctionIds(
-    const std::vector<uint64_t>& function_ids) {
-  if (function_ids.empty() || !timer_durations_.contains(function_ids[0])) {
+void LiveFunctionsDataView::UpdateHistogramWithScopeIds(const std::vector<uint64_t>& scope_ids) {
+  if (scope_ids.empty() || !timer_durations_.contains(scope_ids[0])) {
     app_->ShowHistogram(nullptr, "", orbit_grpc_protos::kInvalidFunctionId);
     return;
   }
-  const uint64_t function_id = function_ids[0];
-  const FunctionInfo& function = functions_.at(function_id);
-  const std::string function_name = orbit_client_data::function_utils::GetDisplayName(function);
-  const std::vector<uint64_t>* timer_durations = &timer_durations_.at(function_id);
-  app_->ShowHistogram(timer_durations, function_name, function_id);
+  const uint64_t scope_id = scope_ids[0];
+
+  std::string function_name = "MANUAL";
```

Comment:
Why not use the name on the manual instrumentation slice?

Reply:
I was wondering the same. If it is really the next PR that is probably fine. But I wonder in general if we should hide this behind a flag for now...

Reply:
Thinking more of this, I really think it should be behind a flag for now. Having manual instrumentation scopes in Orbit on par with dynamic instrumentation (i.e. in live view, histograms, capture view navigation) includes many steps and also some UX considerations (how to distinguish between the types in the Live view). Adding only partial support may feel really unfinished and we should consider not exposing this too early.

Reply:
Yeah, hiding the stuff behind the flag turned out to be an easy task, so I've done this.